site/js/modules/autograder/test/server.js

```diff
@@ -20,13 +20,27 @@ global.fetch = function(url, options = {}) {
     // Create arguments by lexicographically traversing the content.
     let args = {};
     for (const key of Object.keys(content).sort()) {
-        args[key] = content[key]
+        args[key] = content[key];
+
+        // Capitalize Course ID to match test data format.
+        if (key === 'course-id') {
+            args[key] = args[key].toUpperCase();
+        }
+    }
+
+    let files = [];
+    for (const key of options?.body.keys()) {
+        if (key === "content") {
+            continue;
+        }
+
+        files.push(`__DATA_DIR__(${key})`);
```

Comment:
Add a comment about this.

Remember, you want to always comment code that is tricky or strange.
Someone reading this will probably immediately think this is strange.